components/searching-consultant/common/FilterModal.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { StyleSheet, Modal, View } from 'react-native';
+
+import FilterModalContent from '../filters/FilterModalMain';
+
+// FilterModal Component: Renders a modal for filtering content
+const FilterModal = ({ modalVisible, closeModal, filterContent }) => {
```

Comment:
I think you should bring this component into `filters` folder, as it is not used by any other component than `tab/FilterTabs.js`